target/linux/mpc85xx/image/Makefile

```diff
@@ -126,6 +126,23 @@ define Device/ocedo_panda
 endef
 TARGET_DEVICES += ocedo_panda
 
+define Device/ecw7210
+  DEVICE_TITLE := Edge-core ECW7210
+  DEVICE_PACKAGES := uboot-envtools
+  # this u-boot supports fit images, but has issues with gzipped kernels
+  # larger than about 4.3 MB. Thus, we use an uncompressed kernel.
```

Comment:
Could you elaborate on the issues you were experiencing? Maybe we can fix the underlying issue instead.

Reply:
Regarding the old-style DEVICE_TITLE format, it's because I gave up on 19 for a bit, and went back to 17, got the device at least booting there, and then brought my changes forward to 18, then 19. The result is that I didn't pay attention when copying-and-pasting, and didn't see the new format.

As to the 4.3 MB size issue, I was getting `GUNZIP: uncompress, out-of-mem or overwrite error - must RESET board to recover`, see more [here](https://pastebin.com/7Ag5eG0a).  See also the similar-looking [mpc85xx: Use uncompressed kernel on the HiveAP-330](https://github.com/openwrt/openwrt/pull/2162). Based on my reading, it sounds like there's a compile-time option in the u-boot source that hard-caps the kernel image size, and this u-boot just happens to have it set kind of low. The good news is that we have tons of NAND flash, so uncompressed isn't much of an issue.